.github/workflows/slow-checks.yml

```diff
@@ -0,0 +1,46 @@
+name: Slow-Checks
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
+
+  build-matrix:
+    name: Generate Build Matrix
+    uses: ./.github/workflows/matrix-all.yml
+
+  Tests:
+    name: Integration-Tests (Python-${{ matrix.python-version }}, Exasol-${{ matrix.exasol-version}})
+    needs: [ build-matrix ]
+    runs-on: ubuntu-latest
+    # Even though the environment "manual-approval" will be created automatically,
+    # it still needs to be configured to require interactive review.
+    # See project settings on GitHub (Settings / Environments / manual-approval).
+    environment: manual-approval
+    env:
+      GITHUB_TOKEN: ${{ secrets.ALTERNATIVE_GITHUB_TOKEN ||  secrets.GITHUB_TOKEN }}
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.build-matrix.outputs.matrix) }}
+
+    steps:
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.20.0
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Run Tests and Collect Coverage
+        run: poetry run nox -s test:integration -- -- --coverage --db-version ${{ matrix.exasol-version }}
```

Comment:
Will the coverage results be merged with the results for the unit tests?

Reply:
In bucketfs-python we needed to use `--itde-db-version`.
Which one is correct?

Reply:
> Will the coverage results be merged with the results for the unit tests?

yes the results will be merged.


> In bucketfs-python we needed to use --itde-db-version.
Which one is correct?

Depends on what tools and versions of those tools are used

Reply:
>>   In bucketfs-python we needed to use `--itde-db-version`.   Which one is correct?

> Depends on what tools and versions of those tools are used

Can you explain this a bit more? 
Does error-reporting-python use a database at all? 
is --db-version correct here? 
Why does bucketfs-python need to use `--itde-db-version`?

Reply:
>  Why does bucketfs-python need to use --itde-db-version?

Feel free to call me and we have a look at it, but I don't think this is subject of this PR. 
That said happy to investigate your issues/question at hand regarding `bucketfs-python` with yout,  but not in the context and PR of another project.

>   Does error-reporting-python use a database at all?

No, but it does have integration tests

>  is --db-version correct here?

Should be ok, afik if no DB is requested nothing happens with that parameter.
Keeping it aligned with the template will simplify future udpates though.